wsd/Admin.cpp

```diff
@@ -783,6 +790,57 @@ void Admin::cleanupResourceConsumingDocs()
     _model.cleanupResourceConsumingDocs();
 }
 
+void Admin::cleanupLostKits()
+{
+    static std::map<pid_t, std::time_t> mapKitsLost;
+    std::set<pid_t> internalKitPids;
+    std::vector<int> kitPids;
+    int pid;
+    unsigned lostKitsTerminated = 0;
+    size_t gracePeriod = _defDocProcSettings.getCleanupSettings().getLostKitGracePeriod();
+
+    internalKitPids = LOOLWSD::getKitPids();
```

Comment:
Nitpick: it's more readable to declare variables at their first use, if possible. Also, allows for making them `const` where possible. Fewer lines of code and more information on each line = win-win.

Notice also that in the case of heap objects (like vector and set) the performance is much better with a single `const std::set<pid_t> internalKitPids = LOOLWSD::getKitPids();` than by having a declaration and then assignment.  And since there is no gain of having them on two lines, the performance win is an added bonus!

Reply:
Agree.